pom.xml

```diff
@@ -66,6 +66,11 @@
             <type>war</type>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apereo.cas</groupId>
```

Comment:
Does this mean that the shell dependency will be packaged inside the final cas.war? That should not happen.